src/main/java/website/frontrow/board/Unit.java

```diff
@@ -25,7 +25,7 @@
     /**
      * Current location of an entity.
      */
-    @SuppressWarnings("visibilitymodifier") // subclasses have to have access to this variable
+    @SuppressWarnings("visibilitymodifier")
```

Comment:
Again if the checkstyle is fixed, remove the suprression, if it isn't keep the explanation.

<layout